pages/nft/iscn/_iscnId.vue

```diff
@@ -307,12 +308,14 @@ export default class NFTTestMintPage extends Vue {
   }
 
   async checkIsWhitelisted() {
+    logTrackerEvent(this, 'IscnMintNFT', 'CheckIsWhitelisted', this.address, 1);
     const { data } = await this.$axios.get(getWhitelistApi(this.address))
     return data.isWhitelisted;
   }
 
   async getISCNInfo() {
     try {
+      logTrackerEvent(this, 'IscnMintNFT', 'GetISCNInfo', this.iscnId, 1);
```

Comment:
I think no need to log this since this is auto triggered by `mounted`
We should log the `Warning: No URL in ISCN\'s metadata` below though